### ISSUE TITLE
playing ONLINE MULTIPLAYER games after applying the (DisableNvidiaSignatureChecks.reg) to ANOTHER game (SAFE/NOT SAFE)?

### ISSUE THREAD

**Issue by RogueMo92** [Nukem9/dlssg-to-fsr3#124]
is it safe to play online multiplayer games without restoring the (NvidiaSignatureChecks) or do i have to enable/disable it every time i wanna play an online multiplayer game
for example: i have applied this mod to Alan Wake 2 and i wanna play another game that has online multiplayer .... will that said game be affected or will only be affected if these files in the game files?

**Comment by Z4ndyz** [comment]
The check will be skipped but it is safe, that's only nvidia internal check nothing to do with the game
It is not safe to use the MOD itself or changing the .dll as per modifyng game files breaks ToS and/or injection of files triggers anticheat
Tldr. check? ok for any game but mod usage only for singleplayer

**Comment by RogueMo92** [comment]
> The check will be skipped but it is safe, that's only nvidia internal check nothing to do with the game It is not safe to use the MOD itself or changing the .dll as per modifyng game files breaks ToS and/or injection of files triggers anticheat Tldr. check? ok for any game but mod usage only for singleplayer

i don't wanna use it with online multiplayer of course or anything online if that matters but i don't be worrying about it being disabled or enabled  whenever i wanna play an online game

**Comment by Z4ndyz** [comment]
I've been using that skip since dlsstweaks brought put that method and that's been a long time ago and everything was fine multiple games multiple different anticheats it's safe dont worry
